app/models/katello/content_view_version_export_history.rb

```diff
@@ -44,10 +44,14 @@ def self.latest(content_view, destination_server: nil)
             destination_server: destination_server).order(:created_at).last
     end
 
-    def export_type_from_metadata
+    def self.export_type_from_metadata(metadata)
```

Comment:
I wonder if this method should be in a service class somewhere since ImportHistory seems to care about it too

Reply:
Dont think we need a service calls just for this. We could do concerns/mixing and include in both but I don't think this is bad